definitions/divorce/json/CaseEvent.json

```diff
@@ -2155,6 +2155,8 @@
     "DisplayOrder": 23,
     "PreConditionState(s)": "AwaitingDecreeAbsolute",
     "PostConditionState": "DARequested",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/da-requested-respondent-notification",
```

Comment:
this may change based on my  COS comment

Reply:
Changed endpoint.